### PR TITLE
Shutdown and start of the whole cluster (bsc#1126210)

### PR DESCRIPTION
**Pull request by tbazant**
Updated and extended the way and order of nodes during a complete cluster shutdown/start-up.
This update applies to SES6+

### CODE REVIEW

**Comment by tbazant** [comment]
Adding a rendered version.
[ceph-cluster-shutdown_color_en.pdf](https://github.com/SUSE/doc-ses/files/4134724/ceph-cluster-shutdown_color_en.pdf)

File: xml/admin_operating_services.xml [hunk 1]
```diff
@@ -244,31 +244,78 @@ ceph-rbd-mirror@.service
...
+  <procedure>
+   <title>Shutting Down the Whole &ceph; Cluster</title>
+   <step>
+    <para>
+     Shut down or disconnect any clients accessing the cluster.
+    </para>
+   </step>
+   <step>
+    <para>
+     To prevent CRUSH to automatically rebalance the cluster, set the cluster
```
Comment (asettle) [inline xml/admin_operating_services.xml]:
```suggestion
     To prevent CRUSH automatically rebalancing the cluster, set the cluster

Comment (tbazant) [reply xml/admin_operating_services.xml]: copy pasta error...but should this not be "prevent CRUSH from automatically..."?

Comment (asettle) [reply xml/admin_operating_services.xml]: @tbazant either or. "from" is nicer :)

Comment (asettle) [reply xml/admin_operating_services.xml]: I've edited it so you can just add the suggestion :D

Comment (tbazant) [inline xml/admin_operating_services.xml]:
```suggestion
     To prevent CRUSH from automatically rebalancing the cluster, set the cluster
```

**Pull request merged**